registry/metrics.go

```diff
@@ -35,19 +34,19 @@ func NewMetrics() Metrics {
 			Namespace: "flux",
 			Subsystem: "registry",
 			Name:      "fetch_duration_seconds",
-			Help:      "Duration of image metadata fetches, in seconds.",
+			Help:      "Duration of Image metadata fetches, in seconds.",
 			Buckets:   stdprometheus.DefBuckets,
 		}, []string{fluxmetrics.LabelInstanceID, LabelRepository, fluxmetrics.LabelSuccess}),
 		RequestDuration: prometheus.NewHistogramFrom(stdprometheus.HistogramOpts{
 			Namespace: "flux",
 			Subsystem: "registry",
 			Name:      "request_duration_seconds",
-			Help:      "Duration of HTTP requests made in the course of fetching image metadata",
+			Help:      "Duration of HTTP requests made in the course of fetching Image metadata",
 		}, []string{fluxmetrics.LabelInstanceID, LabelRepository, LabelRequestKind, fluxmetrics.LabelSuccess}),
 	}
 }
 
-func (m Metrics) WithInstanceID(instanceID flux.InstanceID) Metrics {
+func (m Metrics) WithInstanceID(instanceID string) Metrics {
```

Comment:
Ok, adding it back in. But I think that having every file directly coupled to this file is a bad idea.

For example, imagine that we wanted to change the definition of what ImageID is, to change it to be a proper object with a CreatedAt field? It's hard, because of all the coupling (it would take too long and probably break things), so you end up creating another thing instead (called ImageDescription).

And then there's circular dependencies too. But anyway. Added.